S4TF_Tutorial_2.ipynb

```diff
@@ -144,7 +144,7 @@
       "metadata": {
         "id": "Q_XrcNGNbejX",
         "colab_type": "code",
-        "outputId": "e38390ed-c137-4490-9e1b-b45a4c1763f2",
+        "outputId": "e9270b8b-c671-4b1a-c8f7-d960b0f9f7e6",
         "colab": {
           "base_uri": "https://localhost:8080/",
           "height": 136
```

Comment:
Near the top, for "Welcome to this Colab where you will train your first Machine Learning model!", I'd not capitalize Machine Learning here (same with anywhere else in this notebook or subsequent notebooks).